.github/workflows/icu_ci.yml

```diff
@@ -32,6 +32,27 @@ jobs:
           ant init;
           ant check;
 
+  # Out of source build with gcc 8 and Wextra.
```

Comment:
Can we use a newer gcc version here? I suspect that "gcc 8" really wants to be the "latest available version of gcc".
https://gcc.gnu.org/releases.html

Reply:
Switched to version 10, in what is now gcc-10-stdlib, this particular job is removed in accordance with comment below.